.gitignore

```diff
@@ -53,7 +53,7 @@ config/branding.yml
 
 # Ignore some of the initializers
 config/initializers/recaptcha.rb
-# config/initializers/devise.rb
+config/initializers/devise.rb
```

Comment:
wow. thanks for catching this.

Reply:
Yeah, it kept overwriting my local version when I pulled a new branch down